build/Packages.props

```diff
@@ -0,0 +1,61 @@
+<Project>
+
+<PropertyGroup>
+    <NuGetPackageVersion>4.8.0-preview1.5185</NuGetPackageVersion>
+
+    <!-- Prodcon-overrideable, using repo-toolset naming convention -->
+    <NuGetBuildTasksVersion>$(NuGetPackageVersion)</NuGetBuildTasksVersion>
+    <NuGetCommandsVersion>$(NuGetPackageVersion)</NuGetCommandsVersion>
+    <NuGetProtocolVersion>$(NuGetPackageVersion)</NuGetProtocolVersion>
+</PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Update="LargeAddressAware" Version="1.0.3" />
+    <PackageReference Update="Microsoft.Build.Build" Version="15.5.180" />
+    <PackageReference Update="Microsoft.Build.Framework" Version="15.5.180" />
```

Comment:
I don't think we should centrally control these; I think we should let the samples directory free.

Reply:
Agreed, esp. given which packages those were. I disabled it entirely for that tree.